lib/checks/visibility/hidden-content.js

```diff
@@ -1,13 +1,16 @@
-let styles = window.getComputedStyle(node);
+const whitelist = ['SCRIPT', 'HEAD', 'TITLE', 'NOSCRIPT', 'STYLE', 'TEMPLATE'];
+if (
```

Comment:
I notice that this style is used elsewhere in the code - but not consistently. I would prefer we stick to the original coding style for if statements and only break lines if they go above 80 characters - and also keep braces and parentheses on the same lines